docs/rules/Community-rules.md

```diff
@@ -0,0 +1,25 @@
+---
+id: Community-rules
+sidebar_label: Community rules
+title: Community rules
+---
+
+Many rules have been developed by the community.
+Follow [this documentation](external-rules.md) to use them in your project.
+
+
+
+## Available rules:
+*[Edit](https://github.com/scalacenter/scalafix/edit/master/docs/developers/cross-publish-custom-rules.md) 
```

Comment:
```suggestion
*[Edit](https://github.com/scalacenter/scalafix/edit/master/docs/rules/Community-rules.md) 
```

Reply:
maybe the file name could be lower case to be consistent with other non-rule doc pages?

Reply:
ok.